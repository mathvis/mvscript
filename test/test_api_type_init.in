let a: vector = Vector(0);
let a: matrix = Matrix([0, 1]);
let a: point = Point(0);
