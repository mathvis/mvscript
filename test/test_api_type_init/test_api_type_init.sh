#!/bin/bash
helix test_api_type_init.mvs test_api_type_init.out --vsplit
