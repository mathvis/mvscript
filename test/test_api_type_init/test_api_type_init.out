Decl (Variable (VarIdentifier "a") (Just vector) (Just (Type (Vector [Type (Int 0)]))))
Decl (Variable (VarIdentifier "a") (Just matrix) (Just (Type (Matrix [Type (Array [Type (Int 0),Type (Int 1)])]))))
Decl (Variable (VarIdentifier "a") (Just point) (Just (Type (Point [Type (Int 0)]))))
